src/client/components/ListOfWeeksWithDetail/index.js

```diff
@@ -0,0 +1,127 @@
+import Alert from "react-bootstrap/Alert";
```

Comment:
Should the new components have snapshot tests?

Reply:
yeah definitely! will add